Makefile

```diff
@@ -350,26 +388,30 @@ add-on: node_modules \
 	      $(built_add_on_shared_js_files) \
 	      $(built_add_on_vendor_js_files) \
 	      $(built_add_on_l10n_files) \
+	      $(built_add_on_module_js_files) \
 	      $(BUILT)/$(ADD-ON)/chrome.manifest \
 	      $(BUILT)/$(ADD-ON)/chrome/locale/chrome.manifest \
 	      $(BUILT)/$(ADD-ON)/install.rdf \
-	      $(BUILT)/add-on/chrome/content/preferences/prefs.js
+	      $(BUILT)/add-on/chrome/content/preferences/prefs.js \
+	      $(BUILT)/add-on/bootstrap.js
 	mkdir -p $(BUILT)/$@
-	$(RSYNC) $@/chrome/bootstrap.js $(BUILT)/$@
 	mkdir -p $(BUILT)/$@/chrome/content/panels
-	$(RSYNC) $@/panels $(BUILT)/$@/chrome/content
-	mkdir -p $(BUILT)/$@/chrome/content/modules
-	$(RSYNC) $@/chrome/modules $(BUILT)/$@/chrome/content
+	$(RSYNC) --exclude='*.js' $@/panels $(BUILT)/$@/chrome/content
+	$(RSYNC) $@/panels/vendor $(BUILT)/$@/chrome/content/panels
 	mkdir -p $(BUILT)/$@/chrome/test
 	$(RSYNC) $@/chrome/test $(BUILT)/$@/chrome
 	mkdir -p $(BUILT)/$@/chrome/content/shared
-	$(RSYNC) shared $(BUILT)/$@/chrome/content
+	$(RSYNC) --exclude='*.js' shared $(BUILT)/$@/chrome/content
+	$(RSYNC) shared/vendor $(BUILT)/$@/chrome/content/shared
 	$(RSYNC) $@/chrome/skin $(BUILT)/$@/chrome/
 
 $(BUILT)/$(ADD-ON)/chrome.manifest: $(ADD-ON)/jar.mn
 	@mkdir -p $(@D)
 	python bin/generateChromeManifest.py --input-file=$^ --output-file=$@
 
+$(BUILT)/$(ADD-ON)/bootstrap.js: $(ADD-ON)/chrome/bootstrap.js
+	$(BABEL) $< --out-file $@
```

Comment:
I was just testing it out, so it wasn't necessary to get things building, but it should be removed because it'll be included in `built_add_on_js_files`.